### PR TITLE
270 Degree Hinge

### PR DESCRIPTION
**Pull request by chrisgonzales**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Adds 270 degrees of swivel to the front panels, along with secure hardware mounting versus tape. Latch and handle provided as well. This should help those that want more range of motion, of like me trouble with slightly larger cut panels that the spec hardware tolerance were too tight for.

#### How was it tested? 
Tested by me, through multiple iterations of modded spec hinges, dual pivot hinges. Then attempt at 270 with tape but didn't have success with my tape solution holding up after few heat cycles, might be a polycarbonate issue. So this last attempt was to do solid mounting, which allows secure installation, but drilling can be made to slots for more freedom of adjustment.

#### Further notes
The design is parametric so other should beable to use the existing STLs or enter new values and export custom size pieces.

### CODE REVIEW

**Comment by chrisgonzales** [comment]
Looks like a commit to the the readme in the same area messed with this if needed I can remerge I'll wait till this has been reviewed but I can fix the conflict right before so I'm not possibly doing it multiple times.

File: printer_mods/chrisrgonzales/270_degree_hinge/README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+faceWidth      : width of facing hinge from edge of extrusion to edge of center of printer. Not actual width of hinge default (34.9 mm)
+
+```
+
+### Questions / Suggestions
+If you have any questions or suggestions feel free to contact me on Discord [chrisrgonzales#0731](https://discord.com/users/chrisrgonzales#0731)
+
+![](Images/installed-close.jpg)  |  ![](Images/installed-180.jpg) |  ![](Images/installed-270.jpg)
+ --- | --- | ---
+ 0° |  180° |  270°
```
Comment (KiloQubit) [inline printer_mods/chrisrgonzales/270_degree_hinge/README.md:121]: For 4 mm and 6 mm you include both drilled and tape STLs, but for 3 mm and 5 mm you only include drilled - should 3 and 5 mm tape versions also be included?

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  All the STLs seem fine, but I have a bunch of comments on the README below.

**Comment by chrisgonzales** [comment]
Thanks for the grammar check one i'm terrible at that, in addition to writing those while it's late haha. Made changes as per suggestion and updated my fork to be up to date as of this post.

If there is anything i missed or any changes i'll get right on those

File: printer_mods/chrisrgonzales/270_degree_hinge/README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+#### Handles
+```
+3 mm x 6 mm magnets  [x4]
+M3 x 6 mm BHCS       [x8]
+M3 Threaded Insert   [x8]
+```
+
+## Printing
+Use the default recommend Voron parts. Can be done in PLA but i have not tested this. If having issues printing side mount
```
[the remark this answers is not in the document]
Comment (KiloQubit) [reply printer_mods/chrisrgonzales/270_degree_hinge/README.md:54]: This is still not fixed.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the update!  There's one thing in the README still not fixed.

File: printer_mods/chrisrgonzales/270_degree_hinge/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (chrisgonzales) [reply printer_mods/chrisrgonzales/270_degree_hinge/README.md]: I apologize i missed that one, committed update.

File: printer_mods/chrisrgonzales/270_degree_hinge/README.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ M3 Threaded Insert   [x8]
 ```
 
 ## Printing
-Use the default recommend Voron parts. Can be done in PLA but i have not tested this. If having issues printing side mount
+Use the default recommended Voron parts. Can be done in PLA but I have not tested this. If having issues printing side mounts
```
Comment (KiloQubit) [inline printer_mods/chrisrgonzales/270_degree_hinge/README.md:54]: Sorry, I missed this before - change `recommended Voron parts` to `recommended for Voron parts`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Sorry, I missed a `for` in that sentence.

File: printer_mods/chrisrgonzales/270_degree_hinge/README.md (hunk 3 above)
Comment (chrisgonzales) [reply printer_mods/chrisrgonzales/270_degree_hinge/README.md]: All good you have a good idea for catching it.

**Review by KiloQubit — APPROVED** [review]
Thanks for all the updates!

**Pull request merged**